.circleci/shellcheck.sh

```diff
@@ -13,7 +13,7 @@ done < <(
     .circleci/phpcs.sh \
     .circleci/shellcheck.sh \
     -type f \
-    \( ! -name "README.*" -and ! -name "_bats-mock.bash" \) \
+    \( ! -name "README.*" -and ! -name "_bats-mock.bash" -and ! -name "govcms-yaml_lint" -and ! -name "*.yml" \) \
```

Comment:
Because govcms-yaml_lint is not really a shell script we can't run shell check over it so we exclude it here.